src/main/java/com/wolApi/wolApi/api/bibleVerses/BibleVerse.java

```diff
@@ -0,0 +1,84 @@
+package com.wolApi.wolApi.api.bibleVerses;
+
+import jakarta.persistence.*;
+
+@Entity
+@Table(name = "bible-verses")
+public class BibleVerse {
```

Comment:
Put this file in an `entities` folder

Reply:
I'm wondering if it would be better to keep it like it is because we have our resources seperated like this -> `api/bibleVerses`

So if we ever add another resource it would just save the hassle of making a new entity folder each time. Hope this makes sense

Reply:
I understand. Typically, if in the future we added more db entities, this is easier. Single Entities aren't tied to a resource endpoint, it's a 1-many